<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>491</width>
      <height>304</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton4">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton7">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton1">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton5">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton8">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton6">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton9">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="newGame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X Goes First!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuColors">
    <property name="title">
     <string>Colors</string>
    </property>
    <addaction name="actionBlue"/>
    <addaction name="actionRed"/>
    <addaction name="actionGreen"/>
    <addaction name="actionGray"/>
   </widget>
   <addaction name="menuColors"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBlue">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
